iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/IcebergConfig.java

```diff
@@ -158,6 +158,15 @@ public class IcebergConfig extends Config implements OverwriteDefaultConfig {
           .stringConf()
           .create();
 
+  public static final ConfigEntry<String> ICEBERG_REST_CATALOG_PROVIDER =
+      new ConfigBuilder(IcebergConstants.ICEBERG_REST_CATALOG_PROVIDER)
+          .doc(
+              "The implementation of IcebergTableOpsProvider defines how the Iceberg REST catalog server gets iceberg catalogs.")
+          .version(ConfigConstants.VERSION_0_6_0)
+          .stringConf()
+          .createWithDefault(
+              "org.apache.gravitino.iceberg.common.ops.ConfigIcebergTableOpsProvider");
```

Comment:
`ConfigIcebergTableOpsProvider` -> `ConfigBasedIcebergTableOpsProvider`?

Reply:
this's an wrong default value, why the test works?

Reply:
You didn't use ICEBERG_REST_CATALOG_PROVIDER to get the provider below,  It's better to use  IcebergConfig
```java
      String className =
          properties.getOrDefault(
              IcebergConstants.ICEBERG_REST_CATALOG_PROVIDER,
              ConfigBasedIcebergTableOpsProvider.class.getName());
```